OrbitLinuxTracing/include/OrbitLinuxTracing/TracingOptions.h

```diff
@@ -0,0 +1,21 @@
+#ifndef ORBITLINUXTRACING_INCLUDE_ORBITLINUXTRACING_TRACINGOPTIONS_H_
```

Comment:
This should be `ORBIT_LINUX_TRACING_TRACING_OPTIONS_H_` for consistency with the other files.

Reply:
Will do that.